### PR TITLE
BUG: Do not impact limit fill price

### PR DESCRIPTION
**Pull request by brianpfink**
Fix for https://github.com/quantopian/qexec/issues/2206

### CODE REVIEW

File: zipline/finance/slippage.py [hunk 1]
```diff
@@ -255,15 +255,17 @@ def process_order(self, event, order):
...
-        simulated_impact = volume_share ** 2 \
-            * math.copysign(self.price_impact, order.direction) \
-            * event.price
+        # if this is a limit order, do not impact the fill price
+        if order.limit:
+            simulated_impact = 0
+        else:
+            simulated_impact = volume_share ** 2 \
+                * math.copysign(self.price_impact, order.direction) \
+                * event.price
```
Comment (jfkirk) [inline zipline/finance/slippage.py]: Would this impact TRAIL orders, or does order.limit exist IFF the order is an LMT order?

Comment (brianpfink) [inline zipline/finance/slippage.py]: we do not support TRAIL orders currently.

**Comment by jfkirk** [comment]
Nope, it is clear, thanks. I think this is good to merge.

Assigning @ehebert for another pair of eyes.

**Comment by brianpfink** [comment]
@jbredeche @ehebert performance looks good to me:
http://performance.dynoquant.com:8060/compareFlights?start=151&end=152

**Pull request merged**